README.md

```diff
@@ -55,9 +55,10 @@ URL Metadata
       ex success:
 
         {
-          error: "",
+          request_error: "",
+          url_errors: {},
```

Comment:
For my own curiosity, should we just merge `urls` and `url_errors` into a single object?
It'd save people having to lookups against two places, and we could just do something like:

```js
{
  "request_error": "",
  "urls": {
    "http://www.example.bad": {
      "error": "Error: Request Failure: 500 Internal"
    },
    "http://www.example.com/good": {
      "url": "http://www.example.com/good",
      "original_url": "http://www.example.com/good",
      "title": "An Example Page",
      ...
    }
  }
}
```

I think this is similar to Embedly's approach, where it returns a `type` of "error", along with a few other keys:

> `error`: When accessing multiple urls at once Embedly will not throw HTTP errors as normal. Instead, it will return an `error` type response that includes the `url`, `error_message` and `error_code`.

> &mdash; via http://docs.embed.ly/docs/extract#response-attributes